aries_cloudagent/config/argparse.py

```diff
@@ -1453,6 +1454,13 @@ def add_arguments(self, parser: ArgumentParser):
             env_var="ACAPY_MULTITENANT_ADMIN",
             help="Specify whether to enable the multitenant admin api.",
         )
+        parser.add_argument(
+            "--multitenancy-config",
+            type=str,
+            metavar="<multitenancy-config>",
+            env_var="ACAPY_MULTITENANCY_CONFIGURATION",
+            help="Specify multitenancy configuration (wallet type and wallet name).",
```

Comment:
An example would be good here. Same as for --wallet-storage-config

Reply:
Added